services/app/src/components/views/quest/cardtemplates/combat/Actions.test.tsx

```diff
@@ -196,6 +196,20 @@ describe('Combat actions', () => {
       const {actions} = runTest({});
       expect(actions[2].node.ctx.templates.combat.mostRecentRolls.length).toEqual(3);
     });
+    test('generates new rolls on new rounds', () => {
```

Comment:
Can you also check that the number of rolls matches the number of adventurers (not players) in a separate test?

Reply:
That's already captured in the test `generates rolls according to player count`